Makefile

```diff
@@ -69,7 +69,7 @@ lint-go:
 lint-git: fetch-git
 	@COMMIT_SHA=`git rev-parse $(OASIS_CORE_GIT_ORIGIN_REMOTE)/$(RELEASE_BRANCH)` && \
 	echo "Running gitlint for commits from $(OASIS_CORE_GIT_ORIGIN_REMOTE)/$(RELEASE_BRANCH) ($${COMMIT_SHA:0:7})..."; \
-	gitlint --commits $(OASIS_CORE_GIT_ORIGIN_REMOTE)/$(RELEASE_BRANCH)...HEAD
+	gitlint --commits $(OASIS_CORE_GIT_ORIGIN_REMOTE)/$(RELEASE_BRANCH)..HEAD
```

Comment:
I'm still curious how/why https://github.com/oasislabs/oasis-core/actions/runs/42614734 worked and https://github.com/oasislabs/oasis-core/actions/runs/43145438 didn't work?

It would be nice if you explained this in git commit's message.

Reply:
> I'm still curious how/why https://github.com/oasislabs/oasis-core/actions/runs/42614734 worked and https://github.com/oasislabs/oasis-core/actions/runs/43145438 didn't work?

Sorry, I see the explanation now: https://github.com/oasislabs/oasis-core/issues/2707#issuecomment-589895264.

> It would be nice if you explained this in git commit's message.

Actually, we should make a note in `.github/workflows/ci-lint.yml` that the Change log fragment check step modifies git history.

Reply:
> Actually, we should make a note in .github/workflows/ci-lint.yml that the Change log fragment check step modifies git history.

I think it's even better if the Change log fragment check step doesn't mess with git's history. I've implemented this in #2712.

Reply:
thanks. I'll back out that part of this PR